src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx

```diff
@@ -56,4 +56,6 @@ While charts are great for reporting on individual pieces, dashboards are the be
 * [Share](/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data#dashboards) your dashboard by email or create a permalink to your dashboard
 * [Import](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-import) or [export](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-json) your dashboard as a JSON, or export your dashboard as a PDF
 * [Duplicate](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#dash-clone) the dashboard to use as a foundation for an entirely new one
-* [Edit](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#add-pages-edit) your dashboard components to add new data streams, modify existing ones, or remove older ones that are no longer relevant
\ No newline at end of file
+* [Edit](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard#add-pages-edit) your dashboard components to add new data streams, modify existing ones, or remove older ones that are no longer relevant
+
+Have you seen our [intro to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) doc and learned [how to use NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/)? If you have, you're ready to start our [guided NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/)!
```

Comment:
I wouldn't use the word 'doc', I think that I'd remove it or write "document"

Reply:
Good call, will remove!